frontend/src/index.scss

```diff
@@ -11,6 +11,7 @@
 @use "components/Carousel/ScrollCarousel";
 @use "components/Navigation/HeaderNav";
 @use "components/Navigation/InnerCopNav";
+@use "components/Navigation/_Footer.scss";
```

Comment:
[Let's import the footer the same way as the above.](https://sass-lang.com/documentation/at-rules/use#partials)